Modules/_zoneinfo.c

```diff
@@ -1709,7 +1709,7 @@ static Py_ssize_t
 parse_abbr(const char *const p, PyObject **abbr)
 {
     const char *ptr = p;
-    char buff = *ptr;
+    char buff;
```

Comment:
I suggest to move the variable declaration closer to where it's used: inside the if block at least.

Reply:
Done